java-checks/src/test/java/org/sonar/java/checks/StaticImportCountCheckTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.checks;
+
+import org.junit.Test;
+import org.sonar.java.checks.verifier.JavaCheckVerifier;
+
+public class StaticImportCountCheckTest {
+
+  private static final String TEST_FILES = "src/test/files/checks/StaticImportCountCheck/";
+
+  @Test
+  public void compliantImports() {
```

Comment:
matter of style, so I don't expect you to change this right away but we tend to like writing explicitely what the test is about using _ as word separator see : https://github.com/SonarSource/sonar-java/blob/20b2b80e521bf0b1b8983a561b9eb0d79fa4ebd5/java-frontend/src/test/java/org/sonar/java/resolve/TypeSubstitutionSolverTest.java#L62